armory/le-04.go

```diff
@@ -13,17 +31,137 @@ func LE_04() (string, pluginkit.TestSetResult) {
 	}
 
 	result.ExecuteTest(LE_04_T01)
+	if result.Tests["LE_04_T01"].Passed {
+		result.ExecuteTest(LE_04_T02)
+		if result.Tests["LE_04_T02"].Passed {
+			result.ExecuteTest(LE_04_T03)
+		}
+	}
```

Comment:
```suggestion
	if result.Tests["LE_04_T01"].Value > 0 {
            result.ExecuteTest(LE_04_T02)
	}
	if result.Tests["LE_04_T02"].Passed {
	    result.ExecuteTest(LE_04_T03)
        }
```

We should continue to test 2 if releases are found. And we don't need to bother nesting the check for whether test 2 passes.